en/docs/learn/tutorials/edit-an-api-by-modifyng-the-api-definition.md

```diff
@@ -0,0 +1,213 @@
+                quantity: 
+                  type: "number"
+                orderId: 
+                  type: "string"
+              title: "Pizza Order"
+        ```
+    
+        !!! note
+            The PUT method has a path parameter defined with reference to the schema "Order".
+            ``` java
+                parameters: 
+                      - 
+                        name: "orderId"
+                        in: "path"
+                        description: "Order Id"
+                        required: true
+                        type: "string"
+                        format: "string"
+                      - 
+                        in: "body"
+                        name: "body"
+                        description: "Order object that needs to be added"
+                        required: true
+                        schema: 
+                          $ref: "#/definitions/Order"
+            ```
+    
+    3. Click **Update Content** .
+       This adds a resource with two HTTP methods into the API, which is visible in the API Publisher 
+       **Resources** tab along with the parameters defined. 
+       ![]({{base_path}}/assets/img/learn/create-rest-api-pizzashack-resources.png)
+
+           Let's assume that the backend of this API sends the response in XML format. Let's document this under the GET method in the resource that we just added.
+    
+        !!! info
+            Troubleshooting:
+               If you get an error after adding the API definition in the Swagger UI, first check the indentation of the code that you added, which defines the API, because Swagger throw errors if the indention is not correct.
+                ![]({{base_path}}/assets/img/learn/rest-api-definition-pizzashack-indentation.png)
+
+7.  Add a summary and description for the GET method.
+
+    1.  Click **Edit Source** and add the following code, which defines a summary and description, to the GET method.
+
+        ``` java
+                    summary: "Get Order details"
+                    description: "Get details of an order by order Id"
+        ```
+
+        ![]({{base_path}}/assets/img/learn/pizzashack-api-get-summary-and-descrption.png)
+
+    2.  Click **Apply Changes** .
+        The summary and description of the GET method that you added is visible when you expand the GET method in the API Publisher.
+        ![]({{base_path}}/assets/img/learn/pizza-shack-api-get-summary-and-description-updated.png)
+
+8.  Change the title of the API.
+
+    1.  Click **Edit Source** , and add the following code in the Swagger UI.
+        This is the title that is visible to the consumers in the Developer Portal after the API is published.
+
+        ``` java
+                    info:
+                      title: PizzaShackAPI
+        ```
+
+        ![]({{base_path}}/assets/img/learn/pizza-shack-api-change-api-title.png)
+        You can see how this change is reflected in the Developer Portal in step 15.
```

Comment:
There are only 12 steps in this doc so mentioning **step 15** here is incorrect.